### ISSUE TITLE
Feature Request: Independent pan per OSC

### ISSUE THREAD

**Issue by baconpaul** [kurasu/surge#304]
I noticed that each OSC doesn't have a pan, so one of my favorite tricks (to widen and narrow pan on a control or gesture between a pair of detuned oscillators) isn't available. Would be great if it was!

**Comment by sense-amr** [comment]
yes this would be an awesome addition too :) just to add sonic capabilities especially considering that if added they could presumably be automated :)

**Comment by sense-amr** [comment]
heheh hold yourself back bacon paul !

**Comment by baconpaul** [comment]
Honestly the biggest problem with implementing this is that there is no place to put the control in the UI!

**Comment by sense-amr** [comment]
i found a place for it :) 
![surge_placeforpan](https://user-images.githubusercontent.com/1504019/51185185-8ad44a00-192a-11e9-8cc0-0e1b82a2b0a3.jpg)

**Comment by baconpaul** [comment]
To make it modulatable easily we need it to be one of the regular sliders (kinda). Also to have the UI be consistent. And when you are in wavetable mode that space gets used for something else doesn't it? But I think you're mostly joking. And anyway after 1.6 we have an idea how to make the layout way more flexible using an SVG file.

But first: zoom.

**Comment by sense-amr** [comment]
Hmm no just checking through all the options for Waves and wavetable that area is always clear and available .. i would think its not a bad place for a pan ..

**Comment by baconpaul** [comment]
Yeah I was thinking maybe a little round knob below the solo and filter routing buttons could work also. But then we need a modulatable knob widget. Which we also don't have (but could write - and it might be useful in other places). The problem with putting it where you say is there's not a natural modulation gesture which fits the rest of the modulation approaches.

But this is for after 1.6 anyway! We can ask @kurasu @cyanit and @itsmedavep for opinions if we add this of where the control could go.

**Comment by baconpaul** [comment]
Link this to #1117

**Comment by baconpaul** [comment]
Link #1150 here also.
